keys.go

```diff
@@ -56,6 +71,42 @@ func (kr *KeysResource) Create(ctx context.Context, ckr CreateKeyRequest) (*Key,
 	return body[Key](kr, req)
 }
 
+// CreateAuthKey creates a new authentication key. Returns the generated [Key] if successful.
+func (kr *KeysResource) CreateAuthKey(ctx context.Context, ckr CreateAuthKeyRequest) (*Key, error) {
```

Comment:
If you take a `CreateKeyRequest` here, you can have the old `Create` method just call the new `CreateAuthKey` method. That makes it clear that they're doing the same thing.